components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/consent/SSOConsentServiceImpl.java

```diff
@@ -417,7 +419,7 @@ private AddReceiptResponse addReceipt(String subject, String subjectTenantDomain
             receiptResponse = getConsentManager().addConsent(receiptInput);
         } catch (ConsentManagementException e) {
             throw new SSOConsentServiceException("Consent receipt error", "Error while adding the consent " +
-                                                                                 "receipt", e);
+                    "receipt", e);
```

Comment:
Shall we move this line to the previous line?

Reply:
Fixed in 06271985d0d22192422630521503f04db25e5eff.